ndscan/plots/image_2d.py

```diff
@@ -137,7 +138,7 @@ def _update(self):
 
             # TODO: Splat old data for progressively less blurry look on refining scans?
             self.image_data = np.full(
-                (_num_points_in_range(x_range), _num_points_in_range(y_range)), 0.0)
+                (_num_points_in_range(x_range), _num_points_in_range(y_range)), np.NaN)
```

Comment:
I didn't know the camel-case version even existed! To cite https://numpy.org/doc/stable/reference/constants.html#numpy.NaN, "use `nan`".